### PR TITLE
Prepares watchmaker v0.21.5 release

### PR DESCRIPTION
**Pull request by kumoy**

### CODE REVIEW

**Comment by kumoy** [comment]
go codebuild go

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+**Summary**:
+
+*   ash-linux-formula
+    - Replace watch with listen to address service restarts
+    - Make state RHEL-07-040560 more resilient when the yum group info is missing
+*   scap-formula
+    - Updates SCAP content from DISA (as of February 2021) and OpenSCAP (v0.1.54)
+*   watchmaker-salt-content
+    - Bump salt version to 2019.2.8
+*   Bump readme-renderer from 28.0 to 29.0
```
Comment (lorengordon) [inline CHANGELOG.md:18]: We can remove this. We don't generally list out changes that are not user-facing. This one is only relevant to the CI/packaging processes...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+### 0.21.5
+
+**Commit Delta**: [Change from 0.21.4 release](https://github.com/plus3it/watchmaker/compare/0.21.4...0.21.5)
+
+**Released**: 2021.02.25
+
+**Summary**:
+
+*   ash-linux-formula
+    - Replace watch with listen to address service restarts
```
Comment (lorengordon) [inline CHANGELOG.md:12]:
```suggestion
    - Replace `watch` requisite with `listen` to restart the `sshd` service a single time
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+    - Make state RHEL-07-040560 more resilient when the yum group info is missing
+*   scap-formula
+    - Updates SCAP content from DISA (as of February 2021) and OpenSCAP (v0.1.54)
+*   watchmaker-salt-content
```
Comment (lorengordon) [inline CHANGELOG.md:16]: We didn't actually modify the `watchmaker-salt-content` submodule. That's a separate repo. We modified the default `config.yaml`, which is part of the watchmaker repo.

```suggestion
*   Update watchmaker default `config.yaml` to use salt v2019.2.8
```

**Review by lorengordon — CHANGES_REQUESTED** [review]
Would like to make a few changes to the changelog...

**Comment by eemperor** [comment]
@kumoy Can you squash the commits into one?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+
+**Summary**:
+
+*   ash-linux-formula
+    - Replace `watch` with `listen` to restart the `sshd` service a single time
+    - Make state RHEL-07-040560 more resilient when the yum group info is missing
+*   scap-formula
+    - Updates SCAP content from DISA (as of February 2021) and OpenSCAP (v0.1.54)
+*   Update watchmaker default `config.yaml` to use salt v2019.2.8
+
```
Comment (eemperor) [inline CHANGELOG.md:17]: Add mention of ability to browse Cloudarmor repo

```suggestion
*    Ability to browse [Watchmaker Cloudarmor repo](https://watchmaker.cloudarmor.io/list.html)
```

**Review by eemperor — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (eemperor) [reply CHANGELOG.md:21]: @kumoy There are some merge stragglers.

**Comment by kumoy** [comment]
go codebuild go

**Review by eemperor — APPROVED** [review]
🍾

**Pull request merged**